crates/biome_cli/src/commands/mod.rs

```diff
@@ -801,13 +801,16 @@ pub(crate) trait CommandRunner: Sized {
             open_uninitialized: true,
         })?;
 
-        workspace.update_settings(UpdateSettingsParams {
+        let result = workspace.update_settings(UpdateSettingsParams {
             project_key,
             workspace_directory: configuration_path.map(BiomePath::from),
             configuration,
             vcs_base_path: vcs_base_path.map(BiomePath::from),
             gitignore_matches,
         })?;
+        for diagnostic in result.diagnostics {
+            console.log(markup! {{PrintDiagnostic::simple(&diagnostic)}});
+        }
```

Comment:
Diagnostic errors should block the CLI. If a plugin is broken or can't be loaded, we risk running operations on files with something the user didn't want to. We already do this if the configuration has errors; we should do the same for plugins.